packages/netlify-cms-core/src/actions/editorialWorkflow.js

```diff
@@ -244,7 +244,10 @@ export function loadUnpublishedEntry(collection, slug) {
         } else {
           dispatch(
             notifSend({
-              message: `Error loading entry: ${error}`,
+              message: {
+                key: 'ui.toast.onFailToLoadEntries',
+                details: error,
+              },
```

Comment:
I can't find these options in the redux-notifications docs, do you have a link for that?

Reply:
no, these are custom properties we send to the view.